src/Nethermind/Nethermind.Trie.Test/PruningScenariosTests.cs

```diff
@@ -62,7 +62,8 @@ public class PruningContext
             [DebuggerStepThrough]
             private PruningContext(TestPruningStrategy pruningStrategy, IPersistenceStrategy persistenceStrategy)
             {
-                _logManager = new TestLogManager(LogLevel.Trace);
+                _logManager = LimboLogs.Instance;
+                    //new TestLogManager(LogLevel.Trace);
```

Comment:
Another one

Reply:
No the tests are currently flooded with messages from those and while tests are fine they are not needed. They would only be useful when we have regression.